.github/workflows/publish.yaml

```diff
@@ -116,4 +116,63 @@ jobs:
           poetry config -- http-basic.pypi $PYPI_USERNAME $PYPI_PASSWORD
           poetry --build publish 
 
+  upload_ersilia_base_to_dockerhub:
+    runs-on: ubuntu-latest
+    needs: version 
+    strategy:
+      matrix:
+        build-version: ["legacy-bentoml", "multistage-condapack"]
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to Docker Hub
+        if: github.event_name != 'pull_request'
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+      - name: Checkout persist credentials
+        uses: actions/checkout@master
+        with:
+          persist-credentials: false 
+          fetch-depth: 0 
+
+      - name: Get version
+        run: |
```

Comment:
Actually we need to keep the _static_version.py file inside the base image since we don't have a way of actually versioning our images right now, since that requires implementing changes in ersilia itself. So let's keep that file instead of using job outputs.